pkg/volume/glusterfs/glusterfs.go

```diff
@@ -966,6 +969,11 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 		case "volumetype":
 			parseVolumeType = v
 
+		case "volumeoptions":
+			if len(v) != 0 {
+				parseVolumeOptions = v
```

Comment:
@rootfs we are not validating any options from provisioner or in heketi. It has been passed to glusterd and validation will only happen on volume creation, because there are atleast 256 volume options available to set on gluster volume.  Validation in any of the above layers will be difficult for the same reason.